MAUI/ImageEditor/zooming.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: Zoom image using .NET MAUI Image Editor Control | Syncfusion
+description: Learn here all about how to zoom image in Syncfusion .NET MAUI Image Editor(SfImageEditor) control and more.
```

Comment:
Description: Learn here all about how to zoom image in the Syncfusion .NET MAUI Image Editor(SfImageEditor) control and more.